rtc-video-quality/encoder_commands.py

```diff
@@ -46,10 +46,20 @@ def rav1e_command(job, temp_dir):
     common_params = [
         '-y',
         '--output', encoded_filename,
-        '--bitrate', job['target_bitrates_kbps'][0],
         clip['y4m_file']
     ]
 
+    if job['param'] == 'bitrate':
+        assert not job['target_bitrates_kbps']
+        param = [
```

Comment:
This is a bit vague. How about something like control_params or sweep_params?

Reply:
Yeah `control_params` is a better variable name. I'll update this 👍